gradle/dependencies.gradle

```diff
@@ -13,15 +13,15 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '161.0.1'
+      mapboxNavigatorVersion = '161.1.0'
```

Comment:
Is there anything to add to the changelog?

Reply:
Yes, they supported ETC 2.0, I added that to changelog and supported it on mapbox-java side as well.